angular-drag-and-drop-lists.js

```diff
@@ -372,16 +381,15 @@
        * one child element per array element.
        */
       element.on('drop', function(event) {
+        // The default behavior in Firefox is to interpret the dropped element as URL and
+        // forward to it. We want to prevent that even if our drop is aborted.
+        event.preventDefault();
```

Comment:
Is there any specific reson to move this ?>

Reply:
Initially appeared to be necessary during testing, but doesn't look to be the case now. Moved it back.